src/normal.rs

```diff
@@ -266,6 +268,26 @@ impl<K: Key, V: Slottable> SlotMap<K, V> {
         self.slots.reserve(needed);
     }
 
+    /// Tries to reserve capacity for at least `additional` more elements to be
+    /// inserted in the `SlotMap`. The collection may reserve more space to
+    /// avoid frequent reallocations.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use slotmap::*;
+    /// let mut sm = SlotMap::new();
+    /// sm.insert("foo");
+    /// sm.try_reserve(32);
```

Comment:
`unwrap()`?

Reply:
Hm. Yes that should have an unwrap. Nice catch.

Reply:
Also, I see there is no `try_set_capacity`?

Reply:
Fixed. Force push.

Reply:
@orlp Why do you expect a `try_set_capacity`? `try_reserve` is trying to set the capacity...

Reply:
Unless you mean `try_reserve_exact`? Requesting for no overcommitment (even though you'll probably still get an overcommit.)

Reply:
I mean for `SecondaryMap` which doesn't have `reserve`.

Reply:
I'm not sure what you mean. Care to rephrase without assumed context?

Reply:
@orlp Okay. I get it. I'll add it.

Reply:
But why isn't it just reserve?

Reply:
Force push'd to include `SecondaryMap::try_set_capacity`.

Reply:
It's not reserve because `reserve` *adds* capacity, while we want to *set* the capacity.

Reply:
@orlp Github does. 

> @james-darkfox james-darkfox force-pushed the james-darkfox:try-reserve branch from 4edb5c2 to ce06cbc 36 minutes ago

Reply:
@james-darkfox Ctrl+F `try_set_capacity` in https://github.com/orlp/slotmap/commit/ce06cbc3409208e35ff502c4c3a9b383bfc08589 gives no results.

Reply:
Whoops. Wrong one

Reply:
@orlp *bump* (github notifications kinda suck...)